### PR TITLE
💂‍♀️ Feature/display-network-QN

### PR DESCRIPTION
**Pull request by stepien4soft**
close: #2275

### CODE REVIEW

**Comment by dmtrjsg** [comment]
Remove "address from the labels"

File: packages/ui/src/app/pages/Settings/Settings.tsx [hunk 1]
```diff
@@ -1,43 +1,75 @@
...
 
 export const Settings = () => {
   const options: NetworkType[] = ['local', 'local-mocks', 'olympia-testnet']
   const [network, setNetwork] = useNetwork()
+  const { t } = useTranslation('settings')
+  const [endpoints] = useNetworkEndpoints()
+  const [currentTab, setCurrentTab] = useState<Tab>('SETTINGS')
+  const tabs = [
+    { title: t('network'), active: currentTab === 'SETTINGS', onClick: () => setCurrentTab('SETTINGS') },
+    { title: t('settings'), active: currentTab === 'LANGUAGE', onClick: () => setCurrentTab('LANGUAGE') },
```
Comment (thesan) [inline packages/ui/src/app/pages/Settings/Settings.tsx:26]: One last details, I think this second tab is supposed to be called language

Comment (stepien4soft) [reply packages/ui/src/app/pages/Settings/Settings.tsx]: done 👍

**Review by thesan — APPROVED** [review]
:+1:

**Pull request merged**